tests/test_itemized.py

```diff
@@ -67,9 +69,34 @@ def test_two_year_transaction_period_default_supplied_automatically(self):
                 two_year_transaction_period=2018
             ),
         ]
+        response = self._response(
+            api.url_for(
+                ScheduleAView,
+                two_year_transaction_period=[2016, 2018],
+        ))
+        self.assertEqual(len(response['results']), 2)
+
+    def test_multiple_two_year_transaction_period_schedule_b(self):
+        """
+        testing schedule_b api can take multiple cyccles now
+        """
+        receipts = [
+            factories.ScheduleBFactory(
+                report_year=2016,
+                two_year_transaction_period=2016
+            ),
+            factories.ScheduleBFactory(
+                report_year=2018,
+                two_year_transaction_period=2018
+            ),
+        ]
+        response = self._response(
+            api.url_for(
+                ScheduleBView,
+                two_year_transaction_period=[2016, 2018],
+        ))
+        self.assertEqual(len(response['results']), 2)
```

Comment:
Thanks for adding tests! It might be more meaningful to add test data outside the provided 2-year periods to make sure the filters are activating.

Reply:
good point. I'll update the test data.

Reply:
everything updated. pls check everything is ok and merge it.